src/screens/OtherProfile/hooks/useOtherProfileScreenHooks.tsx

```diff
@@ -103,6 +124,8 @@ const useOtherProfileScreenHooks = (targetUserProfileId: string, username: strin
   };
 
   const initData = async () => {
+    setUserId(params?.data?.user_id);
+    setOtherProfileData({});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good practice to check if `params?.data?.user_id` exists before setting it with `setUserId()`. Also, resetting `otherProfileData` to an empty object might not be necessary if you're fetching new data immediately after in `fetchOtherProfileData()`. Consider removing this line unless there's a specific reason for it.

```diff
- setUserId(params?.data?.user_id);
- setOtherProfileData({});
+ if (params?.data?.user_id) {
+   setUserId(params?.data?.user_id);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->